NotificationHubs/src/com/windowsazure/messaging/RetryUtil.java

```diff
@@ -72,18 +72,46 @@ public static RetryPolicy getRetryPolicy(RetryOptions options) {
                 }
 
                 if (error instanceof TimeoutException) {
-                	System.out.println("TimeoutException error occurred. Retrying operation. Attempt: " + attempt);
-                	
+                	System.out.println("TimeoutException error occurred. Retrying operation. Attempt: " + attempt); 
```

Comment:
`System.out` is not an appropriate way to be logging. Please remove all added instances of this.